source/kraken/fill_disruption_from_database.h

```diff
@@ -378,6 +396,29 @@ struct DisruptionDatabaseReader {
         FILL_REQUIRED(property, type, std::string)
         FILL_REQUIRED(property, value, std::string)
     }
+
+    template <typename T>
+    void fill_application_pattern(T const_it) {
+        pattern = impact->add_application_patterns();
+        FILL_REQUIRED(pattern, start_date, int32_t)
+        FILL_REQUIRED(pattern, end_date, int32_t)
+        const auto& week = std::bitset<7>{const_it["pattern_weekly_pattern"].template as<std::string>()};
+        auto* week_pattern = pattern->mutable_week_pattern();
+        week_pattern->set_monday(week[6 - navitia::Monday]);
```

Comment:
this seems strange to me, is it possible to make this clearer?

Reply:
For information: In data base we have bit(7) of weekly_pattern from Monday to Sunday.

Reply:
OK